### ISSUE TITLE
Incorrect Mandarin romanizations

### ISSUE THREAD

**Issue by ChristianSi** [tatuylonen/wiktextract#61]
As it goes ... some issues are fixed, but now another one has popped up in the latest dump. In various words, the romanizations of some or all Mandarin translations are now incorrectly set to "zh", while the actual pinyin entry is missing. This concerns most translations of "aunt" ("a parent’s sister or sister-in-law") and some of "box" ("to strike with the fists" and related senses). There are dozens of other entries that have the same issue – I do not think that just "zh" is ever a correct romanization of a Mandarin word.

**Comment by tatuylonen** [comment]
I just fixed this (for at least most Chinese translations).  It changed related code yesterday, and it is possible that other similar issues still remain for other languages.  They should be easy to fix once noticed.  This relates to the language code, e.g., (zh), being included in the formatted text by Wiktionary when it does not match the language name.  I have some kludges for this, but they are not perfect, and I changed them yesterday to make them more maintainable.

The update should be reflected on the web site in about five hours.
